package.json

```diff
@@ -43,46 +43,47 @@
     "react-keyed-flatten-children": "^1.3.0"
   },
   "devDependencies": {
-    "@babel/cli": "^7.12.10",
-    "@babel/core": "^7.12.10",
-    "@babel/preset-env": "^7.12.10",
-    "@babel/preset-react": "^7.12.10",
-    "@babel/register": "^7.12.10",
+    "@babel/cli": "^7.12.13",
+    "@babel/core": "^7.12.13",
+    "@babel/plugin-transform-spread": "7.12.1",
+    "@babel/preset-env": "^7.12.13",
+    "@babel/preset-react": "^7.12.13",
+    "@babel/register": "^7.12.13",
     "@visionappscz/eslint-config-visionapps": "^1.3.1",
     "@visionappscz/stylelint-config-visionapps": "^1.1.0",
     "@visionappscz/stylelint-config-visionapps-order": "^1.1.0",
-    "autoprefixer": "^10.1.0",
+    "autoprefixer": "^10.2.4",
     "babel-core": "^7.0.0-bridge.0",
     "babel-eslint": "^10.1.0",
     "babel-jest": "^26.6.3",
     "babel-loader": "^8.2.2",
     "concurrently": "^5.3.0",
-    "core-js": "^3.8.1",
+    "core-js": "^3.8.3",
     "css-loader": "^5.0.1",
     "enzyme": "^3.11.0",
-    "enzyme-adapter-react-16": "^1.15.5",
+    "enzyme-adapter-react-16": "^1.15.6",
     "enzyme-to-json": "^3.6.1",
-    "eslint": "^7.15.0",
+    "eslint": "^7.19.0",
     "eslint-config-airbnb": "^18.2.1",
     "eslint-loader": "^4.0.2",
     "eslint-plugin-import": "^2.22.1",
     "eslint-plugin-jsx-a11y": "^6.4.1",
     "eslint-plugin-markdown": "^2.0.0-rc.0",
     "eslint-plugin-mdx": "^1.8.2",
-    "eslint-plugin-react": "^7.21.5",
+    "eslint-plugin-react": "^7.22.0",
     "eslint-plugin-react-hooks": "^4.2.0",
     "identity-obj-proxy": "^3.0.0",
     "jest": "^26.6.3",
-    "markdownlint-cli2": "0.0.11",
-    "nodemon": "^2.0.6",
-    "postcss": "^8.2.1",
-    "postcss-loader": "^4.1.0",
+    "markdownlint-cli2": "^0.0.13",
```

Comment:
Let's be careful with 0.x versions, I'd leave out the `^`.

Reply:
Yep, I overlooked this. Thx